doc/README.md

```diff
@@ -65,7 +65,7 @@ make linkcheck
 To run tests for examples shipping with docstrings in Python files, run the following command:
 
 ```shell
-RAY_MOCK_MODULES=0 make doctest
+make doctest
```

Comment:
We should remove this section or replace it with the information in [How to write code snippets](https://docs.ray.io/en/master/ray-contribute/writing-code-snippets.html#how-to-locally-test-examples). We don't use `make doctest` anymore.